offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/OfferToPDFConverter.groovy

```diff
@@ -371,6 +374,38 @@ class OfferToPDFConverter implements OfferExporter {
                    """
         }
 
+        static String subTableFooter(String productGroup){
+
+            String footerTitle = ""
+
+            if (productGroup == "dataGeneration"){
+                footerTitle = "Data Generation"
+            }
+            if (productGroup == "dataAnalysis"){
+                footerTitle = "Data Analysis"
+            }
+            if (productGroup == "dataManagement"){
+                footerTitle = "Data Management"
+            }
+
```

Comment:
Maybe you can use an enum instead of a string. Then you can remove the ifs here and just get the string value from the enum value that was provided. Just as you did it for some other PR you worked on, do you remember?

Reply:
It is done! 👍